recipes/pixy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pixy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksamuk/pixy/archive/{{ version }}.tar.gz
+  sha256: 40cb4db24d08d86347fe2a6dd64bd2e20f8a9c9671f19202086e5c72b670a776
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

Comment:
you can combine these as `skip: True  # [win or py<3]`

Reply:
Thanks for your help with this!